app.js

```diff
@@ -11,3 +11,171 @@ document.querySelectorAll('.nav-link').forEach((n) => n.addEventListener('click'
   document.querySelector('body').classList.toggle('active');
 }));
+
+const cards = [
+  {
+    name: 'Tonic',
+    company: 'CANOPY',
+    stack: 'Back End Dev',
+    year: '2015',
+    description: `
+    A daily selection of privately personalized reads; no accounts or
+    sign-ups required.
+    `,
+    featuredImage: '/images/Snapshoot-Portfolio.png',
+    tech1: 'html',
+    tech2: 'css',
+    tech3: 'javascript',
+    modalId: 'card1',
+    liveVersionLink: 'https://github.com/hadi111jafari',
+    sourceLink: 'https://github.com/hadi111jafari',
+  },
+
+  {
+    name: 'Multi-Post Stories',
+    company: 'CANOPY',
+    stack: 'Back End Dev',
+    year: '2015',
+    description: `
+    Experimental content creation feature that allows users to add to an
+    existing story over the course of a day without spamming their
+    friends.
+    `,
+    featuredImage: '/images/Snapshoot-Portfolio1.png',
+    tech1: 'html',
+    tech2: 'css',
+    tech3: 'javascript',
+    modalId: 'card2',
+    liveVersionLink: 'https://github.com/hadi111jafari',
+    sourceLink: 'https://github.com/hadi111jafari',
+  },
+
+  {
+    name: 'Facebook 360',
+    company: 'CANOPY',
+    stack: 'Back End Dev',
+    year: '2015',
+    description: `
+    Exploring the future of media in Facebook's first Virtual Reality
+    app; a place to discover and enjoy 360 photos and videos on Gear VR. `,
+    featuredImage: '/images/Snapshoot-Portfolio2.png',
+    tech1: 'html',
+    tech2: 'css',
+    tech3: 'javascript',
+    modalId: 'card3',
+    liveVersionLink: 'https://github.com/hadi111jafari',
+    sourceLink: 'https://github.com/hadi111jafari',
+  },
+  {
+    name: 'Uber Navigation',
+    company: 'CANOPY',
+    stack: 'Back End Dev',
+    year: '2015',
+    description: `
+    A smart assistant to make driving more safe, efficient, and fun by
+    unlocking your most expensive computer: your car. `,
+    featuredImage: '/images/Snapshoot-Portfolio2.png',
+    tech1: 'html',
+    tech2: 'css',
+    tech3: 'javascript',
+    modalId: 'card4',
+    liveVersionLink: 'https://github.com/hadi111jafari',
+    sourceLink: 'https://github.com/hadi111jafari',
+  },
+];
+
+function newCard(card) {
+  return `
+    <div class="card">
+        <img src="${card.featuredImage}" alt="" />
+        <div class="card-content">
+          <h2>${card.name}</h2>
+          <div class="list-1">
+            <p>${card.company}</p>
+            <ul class="field">
+              <li class="backend">${card.stack}</li>
+              <li class="year">${card.year}</li>
+            </ul>
+          </div>
+
+          <p>
+          ${card.description}
+          </p>
+          <ul class="languages">
+            <li>${card.tech1}</li>
+            <li>${card.tech2}</li>
+            <li>${card.tech3}</li>
+          </ul>
+          <div id="btn-curser">
+          <button onclick="openModal('${card.modalId}')" class="btn modal-btn" id="${card.modalId}">See Project</button>
```

Comment:
- [ ] The user needs to click twice on the button to see the pop-up. Kindly fix this issue and make sure that the pop-up shows up after one click.